src/analysis/interproc/interproc.rs

```diff
@@ -66,15 +65,11 @@ impl<'a,'b, M, T> InterProcAnalyzer<'a, 'b, M, T>
 #[cfg(test)]
 mod test {
     use super::*;
-    use frontend::source::{FileSource, Source};
```

Comment:
It uses `Source`. Without importing `Source` this trait will not be implemented on `r2::R2`.